src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/spreadsheet/PricesSpreadsheetGenerator.kt

```diff
@@ -32,10 +30,14 @@ class PricesSpreadsheetGenerator(
   internal fun generate(supplier: Supplier, startDate: LocalDate): File {
     val dateGenerated = timeSource.date()
 
-    XSSFWorkbook(template.get()).use { workbook ->
+    SXSSFWorkbook().use { workbook ->
       val header = PriceSheet.Header(dateGenerated, ClosedRangeLocalDate(startDate, endOfMonth(startDate)), supplier)
+
       val moves = moveService.moves(supplier, startDate)
 
+      SummarySheet(workbook, header).also { logger.info("Adding summaries.") }
```

Comment:
Moved up here as the order of these dictates the position of the sheets in the generated spreadsheet.